docs/source/migration_guides/coming_from_dask.md

```diff
@@ -0,0 +1,122 @@
+# Dask Migration Guide
+
+This migration guide explains the most important points that anyone familiar with Dask should know when trying out Daft or migrating Dask workloads to Daft. The guide includes an overview of technical,conceptual and syntax differences between the two libraries that you should be aware of. Understanding these differences will help you evaluate your choice of tooling and ease your migration from Dask to Daft.
+
+## When should I use Daft?
+
+Dask and Daft are DataFrame frameworks built for distributed computing. Both libraries enable you to process large, tabular datasets in parallel, either locally or on remote instances on-prem or in the cloud.
+
+If you are currently using Dask, you may want to consider migrating to Daft if you:
+
```

Comment:
Another point is that we have deep support for data catalogs, predicate pushdowns and metadata pruning from Iceberg, Delta, and Hudi